docs/internal/release/release-process.md

```diff
@@ -0,0 +1,37 @@
+# Release process.
+
+## Regular release.
+
+This is a regular flow, when you are just following our usual versioning and there no additional conditions.
+
+For a `next` release, run Release workflow from Github actions panel on `development` branch.
+For a stable release, do the following:
+
+1. Create branch from `development` for next minor/major as `release/v1.2.x`.
+2. Update `libs/lerna.json` version to a new minor/major version as `1.2.0`.
+3. Push the release branch.
+4. Trigger Release workflow from Github actions panel on release branch.
+   This creates an RC release.
+5. On `development` branch update `libs/lerna.json` version to a next minor version as `1.3.0`
+
+For a stable release on a release branch, do the following:
+
+1. Update `libs/lerna.json` version to remove RC tag `1.2.0-rc.4` -> `1.2.0`.
+2. Push the version to the release branch.
+3. Trigger Release workflow from Github actions panel on release branch.
+
+For a patch on a release branch, do the following:
+
+1. Push fixes to the release branch.
```

Comment:
Will the version be automatically updated in this case? (1.2.0 -> 1.2.1 etc.)

Reply:
Yep.